.github/workflows/ci.yml

```diff
@@ -103,28 +103,20 @@ jobs:
   build_test_all_windows:
     needs: setup
     if: contains(fromJson(needs.setup.outputs.enabled-jobs), 'build_test_all_windows')
-    runs-on: windows-2022-64core
+    # GitHub-hosted 64 core runners are too expensive and take too long to check out the repo
+    # TODO(scotttodd): disable GitHub-hosted if too slow
+    # TODO(saienduri): switch to self-hosted
+    runs-on: windows-2022
```

Comment:
Got my hopes up with https://github.blog/2024-01-17-github-hosted-runners-double-the-power-for-open-source/ . Still looks to be very slow: https://github.com/iree-org/iree/actions/runs/8806346639/job/24170965376?pr=17145. Will probably have to disable the `build_test_all_windows` job entirely until we get a self-hosted Windows runner capable of building in < 20 minutes.

I doubt ccache will help much. Could also update the `CCACHE_NAMESPACE: github-windows-2022-64core` and run some experiments...